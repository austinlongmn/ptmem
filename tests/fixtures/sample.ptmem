# Mathematics

- What is 2 + 2?
+ 4

- What is the square root of 16?
+ 4

- What are the first three prime numbers?
+ 2
+ 3
+ 5

# Science

- What is the chemical symbol for water?
+ H2O

- What gas do plants absorb during photosynthesis?
+ Carbon dioxide
+ CO2

- Who developed the theory of relativity?
+ Albert Einstein

# History

- When did World War II end?
+ 1945

- Who was the first president of the United States?
+ George Washington

- In what year did the Berlin Wall fall?
+ 1989

# Geography

- What is the capital of France?
- What city is the capital of France?
+ Paris

- Which is the largest ocean?
+ Pacific Ocean

- What are the seven continents?
+ Asia
+ Africa
+ North America
+ South America
+ Antarctica
+ Europe
+ Australia

/ This is a comment that should be ignored
/ Comments start with forward slash and space

# Literature

/ Another comment here
- Who wrote "Romeo and Juliet"?
+ William Shakespeare

- What is the first book in the Harry Potter series?
+ Harry Potter and the Philosopher's Stone
+ Harry Potter and the Sorcerer's Stone

# Programming

- What does CPU stand for?
+ Central Processing Unit

- What programming language is known for "write once, run anywhere"?
+ Java

- What does HTML stand for?
+ HyperText Markup Language
