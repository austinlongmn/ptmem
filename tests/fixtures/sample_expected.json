[
    {
        "questions": [
            "What is 2 + 2?"
        ],
        "answers": [
            "4"
        ],
        "category": "Mathematics"
    },
    {
        "questions": [
            "What is the square root of 16?"
        ],
        "answers": [
            "4"
        ],
        "category": "Mathematics"
    },
    {
        "questions": [
            "What are the first three prime numbers?"
        ],
        "answers": [
            "2",
            "3",
            "5"
        ],
        "category": "Mathematics"
    },
    {
        "questions": [
            "What is the chemical symbol for water?"
        ],
        "answers": [
            "H2O"
        ],
        "category": "Science"
    },
    {
        "questions": [
            "What gas do plants absorb during photosynthesis?"
        ],
        "answers": [
            "Carbon dioxide",
            "CO2"
        ],
        "category": "Science"
    },
    {
        "questions": [
            "Who developed the theory of relativity?"
        ],
        "answers": [
            "Albert Einstein"
        ],
        "category": "Science"
    },
    {
        "questions": [
            "When did World War II end?"
        ],
        "answers": [
            "1945"
        ],
        "category": "History"
    },
    {
        "questions": [
            "Who was the first president of the United States?"
        ],
        "answers": [
            "George Washington"
        ],
        "category": "History"
    },
    {
        "questions": [
            "In what year did the Berlin Wall fall?"
        ],
        "answers": [
            "1989"
        ],
        "category": "History"
    },
    {
        "questions": [
            "What is the capital of France?",
            "What city is the capital of France?"
        ],
        "answers": [
            "Paris"
        ],
        "category": "Geography"
    },
    {
        "questions": [
            "Which is the largest ocean?"
        ],
        "answers": [
            "Pacific Ocean"
        ],
        "category": "Geography"
    },
    {
        "questions": [
            "What are the seven continents?"
        ],
        "answers": [
            "Asia",
            "Africa",
            "North America",
            "South America",
            "Antarctica",
            "Europe",
            "Australia"
        ],
        "category": "Geography"
    },
    {
        "questions": [
            "Who wrote \"Romeo and Juliet\"?"
        ],
        "answers": [
            "William Shakespeare"
        ],
        "category": "Literature"
    },
    {
        "questions": [
            "What is the first book in the Harry Potter series?"
        ],
        "answers": [
            "Harry Potter and the Philosopher's Stone",
            "Harry Potter and the Sorcerer's Stone"
        ],
        "category": "Literature"
    },
    {
        "questions": [
            "What does CPU stand for?"
        ],
        "answers": [
            "Central Processing Unit"
        ],
        "category": "Programming"
    },
    {
        "questions": [
            "What programming language is known for \"write once, run anywhere\"?"
        ],
        "answers": [
            "Java"
        ],
        "category": "Programming"
    },
    {
        "questions": [
            "What does HTML stand for?"
        ],
        "answers": [
            "HyperText Markup Language"
        ],
        "category": "Programming"
    }
]
